### PR TITLE
e2e-metal-ipi periodic nightly tests

### PR DESCRIPTION
**Pull request by akiselev1**
Enabling periodic nightly tests for baremetal installer.

### CODE REVIEW

File: ci-operator/config/openshift/installer/openshift-installer-master.yaml [hunk 1]
```diff
@@ -202,3 +202,8 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-nightly
```
Comment (stbenjam) [inline ci-operator/config/openshift/installer/openshift-installer-master.yaml]: Are there other jobs like this for other platforms? I don't see them (but this repo is YAML soup so I might be missing it)

Release jobs would be more valuable in the periodics under `./ci-operator/jobs/openshift/release`

File: ci-operator/config/openshift/installer/openshift-installer-master.yaml [hunk 2]
```diff
@@ -202,3 +202,8 @@ tests:
...
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
+  cron: "0 23 * * *"
```
Comment (stbenjam) [inline ci-operator/config/openshift/installer/openshift-installer-master.yaml]: Could we change this to be more frequent? Say, once every 90 minutes?

See: https://github.com/openshift/release/pull/7869#issuecomment-602975009

Comment (akiselev1) [reply ci-operator/config/openshift/installer/openshift-installer-master.yaml]: 90 min can not be done in one crontab line. Can we go with 1 or 2 hours instead? I would also change job name from nightly to hourly (or just periodic) in this case.

**Comment by akiselev1** [comment]
Dropping periodic approach in favour of release gating.

**Pull request closed without merging**